packages/app/src/explorer/EntityItem.tsx

```diff
@@ -43,6 +46,12 @@ function EntityItem(props: Props) {
     }
   }, [entity, path, selectedPath, toggleExpanded]);
 
+  useEffect(() => {
+    if (isSelected) {
+      btnRef?.current?.focus();
+    }
+  }, [btnRef, isSelected]);
```

Comment:
This is to restore the focus on the selected element when tabbing from search tab to the explorer tab.